src/manager/Manager.sol

```diff
@@ -122,6 +124,28 @@ contract Manager is IManager, VersionedContract, UUPS, Ownable, ManagerStorageV1
         emit DAODeployed({ token: token, metadata: metadata, auction: auction, treasury: treasury, governor: governor });
     }
 
+    /// @notice Set a new metadata renderer
+    /// @param _newRendererImpl new renderer address to use
+    /// @param _setupRenderer data to setup new renderer with
+    function setMetadataRenderer(
+        address _token,
+        address _newRendererImpl,
+        bytes memory _setupRenderer
+    ) external returns (address metadata) {
+        if (msg.sender != IOwnable(_token).owner()) revert ONLY_TOKEN_OWNER();
```

Comment:
nit: it's nice to always use `{` and `}` brackets to prevent bugs and readability.